client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HttpClient.ts

```diff
@@ -25,7 +25,7 @@ export class HttpClient implements IHttpClient {
 
             xhr.open(method, url, true);
             xhr.setRequestHeader("X-Requested-With", "XMLHttpRequest");
-
```

Comment:
The version should be generated automatically based on https://github.com/aspnet/SignalR/blob/7b0be52c545bc20f7402ed1b58c5ff8f99c2030f/version.xml#L6 
(This is not for alpha - I will open an issue for this)

Reply:
https://github.com/aspnet/SignalR/issues/763